### DISCUSSION TITLE
Two laptops connected to one modem running jamulus

### DISCUSSION THREAD

**Discussion by Annehaaij** [jamulussoftware/jamulus#1545] [Bug Investigation]
Good afternoon everyone, 
Our choir is rehearsing with jamulus happily and succesfully! However, one couple is singing from their home, using two laptops, cable-connected to the same modem (500 download, 60 upload, ping 10)When both try to run jamulus at the same time (at the rehearsal) one computer shows red buffer lights and delays high in the hundreds and is not able to run jamulus. The other runs jamulus but seems affected at times (screen suddenly turns black, very high delay, kicked out of jamulus). Does anyone have experience with two computers connected to the same modem and running jamulus at the same time?

**Comment by ghost** [comment]
What is probably happening is that the Jamulus clients on separate computers have the same IP addr after NAT on the modems. The clients also have the same port number because they are on separate computers and choose the same port number. This makes the packets look like they come from a single source (both having the same saddr:port and both the same daddr:port).

The solution is to use the --port <port number> option on the command line to give the clients different port numbers.

ie. Computer1(Jamulus --port 22130) and Computer2(Jamulus --port 22131)

**Comment by Annehaaij** [comment]
Thank you so much! We are going to try this!!

Op di 20 apr. 2021 19:49 schreef DavidSavinkoff ***@***.***>:

**Reply by ghost** [reply]
Hi @Annehaaij, did you resolve the "Two laptops connected to one modem running jamulus" problem?

**Comment by Annehaaij** [comment]
Hi David, the elderly couple that has the 2 computer problem is going to
try it with me next week. I'll let you know as soon as we'be tried it.
Sorry, it takes so long. Anne


Op di 27 apr. 2021 20:25 schreef DavidSavinkoff ***@***.***>:
